pages/management/ManagementHandler.inc.php

```diff
@@ -216,15 +226,15 @@ function workflow($args, $request) {
 		$reviewSetupForm = new \PKP\components\forms\context\PKPReviewSetupForm($contextApiUrl, $locales, $context);
 
 		$emailTemplatesListPanel = new \APP\components\listPanels\EmailTemplatesListPanel(
-			'emailTemplates',
-			__('manager.emails.emailTemplates'),
-			[
-				'apiUrl' => $emailTemplatesApiUrl,
-				'count' => 100,
-				'items' => [],
-				'itemsMax' => 0,
-				'lazyLoad' => true,
-			]
+				'emailTemplates',
```

Comment:
While I'm stuck on nitpicky spacing issues :) we don't generally double-indent in PHP.

Reply:
I have ununintendified this